packages/sanity-plugin-nrkno-spellcheck/src/lib/ui/components/SpellcheckButtonTooltips.tsx

```diff
@@ -2,7 +2,7 @@ import React, { PropsWithChildren, useCallback, useContext } from 'react';
 import { Box, Button, Card, Flex, Label, Popover, Stack, Tooltip } from '@sanity/ui';
 import { SpellcheckDispatch } from './SpellcheckContext';
 import { SpellcheckProgressbar } from './SpellcheckProgressbar';
-import { TooltipProps } from '@sanity/ui/lib/src/primitives/tooltip/tooltip';
+import { TooltipProps } from '@sanity/ui/lib/dts/primitives/tooltip/tooltip';
```

Comment:
Kunne vi brukt 
```suggestion
import type { TooltipProps } from '@sanity/ui/lib/dts/primitives/tooltip/tooltip';
```

Her?